packages/opticss/src/util/cssIntrospection.ts

```diff
@@ -53,17 +51,6 @@ export function eachSelectorIdent(sel: ParsedSelector, idents: IdentTypes, cb: (
   });
 }
 
-/**
- * Iterate over all identifiers (ex: .class and #id) in a given postcss tree.
- * @param root Postcss root to walk.
- * @param cache SelectorCache to ensure we are working with the same selector instances.
- * @param idents Object of type IdentTypes to indicate idents to traverse over.
- * @param cb Callback function passed the identifier, rule and parsed selector.
- */
-export function eachIdent(root: postcss.Root, cache: SelectorCache, idents: IdentTypes, cb: (node: IdentNode, rule: postcss.Rule, sel: ParsedSelector) => void) {
```

Comment:
Below are more unused utility methods. I think they're here for completness' sake. I'll leave it up to you if I unit test or remove them.